source/includes/write/gridfs.php

```diff
@@ -0,0 +1,93 @@
+<?php
+require 'vendor/autoload.php'; // include Composer's autoloader
+
+use MongoDB\Client;
```

Comment:
I missed this earlier. The code below uses `new MongoDB\Client(...)` so this `use` statement has no effect. Consider removing it.

Alternatively, decide if you want to rely on `use` statements for all class references in the interest of consistency. I see you're doing that for `new ObjectId(...)`.